phpBB/composer.lock

```diff
@@ -1977,12 +2022,12 @@
             "version": "v1.0.2",
             "source": {
                 "type": "git",
-                "url": "https://github.com/silexphp/Pimple.git",
```

Comment:
I really don't know... especially that this change shouldn't happen in this direction...